src/extension.ts

```diff
@@ -207,7 +211,7 @@ export async function activate(context): Promise<extensionapi.ExtensionAPI> {
     ];
 
     await azureclusterprovider.init(clusterProviderRegistry, { shell: shell, fs: fs });
-
+    await minikubeclusterprovider.init(clusterProviderRegistry, { shell: shell, createExec: null });
```

Comment:
The TypeScript compiler is failing on this because it doesn't like object literals having extra fields.  You need to remove the `createExec` field.

Reply:
fixed.